api/src/main/java/org/openmrs/module/procedure/Procedure.java

```diff
@@ -23,7 +23,7 @@
  * performed by a healthcare professional, a service provider, a friend or relative or in some cases
  * by the patient themselves.
  */
-public class Procedure extends BaseOpenmrsData implements Serializable {
+public class Procedure extends BaseChangeableOpenmrsMetadata implements Serializable {
```

Comment:
i wanted to get access to some of the methods provided by BaseChangeableOpenmrsMetadata like setRetired(), setDateRetired(),setRetireReason() to be used in deleting a procedure